recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "2.5.0" %}
+{% set version = "2.5.1_RC1" %}
```

Comment:
version 2.5.1_RC1 ??

the release candidates are not welcome on conda-forge! Do not merge before changing it to a definitive version.